src/recoil/graphs/graphs.ts

```diff
@@ -20,3 +20,10 @@ export const graphsAtom = atom<Graph[]>({
     key: 'graphsAtomKey',
     default: graphsDefaultSelector,
 });
+
+export const graphItemSelector = selectorFamily<Graph, string>({
```

Comment:
can't we just call it graphSelector?

item is unnecessary to the name in my opinion